src/main/java/appeng/crafting/v2/CraftingJobV2.java

```diff
@@ -108,6 +108,10 @@ public static CraftingJobV2 deserialize(World world, ByteBuf buffer) {
         return job;
     }
 
+    public CraftingMode getCraftingMode() {
```

Comment:
This is an override right? So should have `@Override` annotated

Reply:
done.